src/wallet/wallet.cpp

```diff
@@ -6331,6 +6331,9 @@ NoteFilter NoteFilter::ForPaymentAddresses(const std::vector<libzcash::PaymentAd
             [&](const libzcash::UnifiedAddress& uaddr) {
                 for (auto& receiver : uaddr) {
                     std::visit(match {
+                        [&](const libzcash::OrchardRawAddress &addr) {
+                            addrs.orchardAddresses.insert(addr);
+                        },
```

Comment:
This conflicts with the same change in #5665 (because in my PR I have the `&` on the other side of the whitespace, because C++ is a wonderful language). I'll rebase that PR after merging this one.